<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>484</width>
    <height>522</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>注册页面</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog {
    background: qlineargradient(
        x1: 0, y1: 0,
        x2: 0, y2: 1,
        stop: 0 #6a11cb, /* 紫色 */
        stop: 1 #2bd7d7 /* 蓝色 */
    );
    border: none;
}</string>
  </property>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>200</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>270</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::EchoMode::Password</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_3">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>340</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::EchoMode::Password</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>210</y>
     <width>54</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>可爱小猫的恋爱警报</family>
     <pointsize>14</pointsize>
     <fontweight>Medium</fontweight>
    </font>
   </property>
   <property name="text">
    <string>账户</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>280</y>
     <width>54</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>可爱小猫的恋爱警报</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>密码</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>350</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>可爱小猫的恋爱警报</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>确认密码</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>90</y>
     <width>311</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>可爱小猫的恋爱警报</family>
     <pointsize>48</pointsize>
     <fontweight>Medium</fontweight>
    </font>
   </property>
   <property name="text">
    <string>注册账户</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>410</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>可爱小猫的恋爱警报</family>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    color: white; /* 文字颜色 */
    background-color: rgba(255, 255, 255, 0.3); /* 背景颜色，带有一定的透明度 */
    border: 2px solid rgba(255, 255, 255, 0.5); /* 边框颜色及宽度，同样带有透明度 */
    border-radius: 10px; /* 圆角程度 */
    padding: 8px 16px; /* 内边距 */
    font-size: 16px; /* 字体大小 */
    backdrop-filter: blur(4px); /* 模糊背景，创造玻璃效果，注意：此属性在某些平台或版本可能不完全支持 */
}

QPushButton:hover {
    background-color: rgba(255, 255, 255, 0.5); /* 鼠标悬停时的背景颜色 */
}

QPushButton:pressed {
    background-color: rgba(255, 255, 255, 0.7); /* 按下按钮时的背景颜色 */
    border-color: rgba(255, 255, 255, 0.8); /* 按下按钮时的边框颜色 */
}</string>
   </property>
   <property name="text">
    <string>注册</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
